problems/removeNumbersAtOddIndices.js

```diff
@@ -2,7 +2,10 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
+const removeNumbersAtOddIndices = (arr) => {
+    for(let i = 0; i < arr.length; i++){
```

Comment:
Try using .filter for cleaner code. Remember the callback on array methods can also take the index as a second argument
We typically don't want to mutate arrays with splice while iterating through them.